### ISSUE TITLE
Broken on current version of Arma

### ISSUE THREAD

**Issue by Kapppa** [gruppe-adler/grad_mtg#8]
On the latest update to Arma 3 this tool no longer works correctly. It produces incorrect files varying from resolution to the correct tile location. 

This is the command I ran.
`gradMtgStart [1, 1];`

This was the Output of 1/0/0.png
[Output](https://cdn.discordapp.com/attachments/471753974643425281/791648899927375882/medium.png)
Expected output:
[Expected](https://maps.gruppe-adler.de/Stratis/sat/1/0/0.png)

Version of Arma 2.00 on Stratis. This was tested on 3 computers with all uiScales.

**Comment by DerZade** [comment]
Hey,

Firstly, please note that grad_mtg is no longer in active development. We moved on to [grad_meh](https://github.com/gruppe-adler/grad_meh) for our internal projects. I just haven't marked this repository as archived yet, because there is no stable release of the meh-project yet (although the current version is a release candidate).

Secondly, the tile you listed as "Expected" is not exported with grad_mtg but with the above-mentioned grad_meh and so are all other tiles currently available on `https://maps.gruppe-adler.de`. `maps.gruppe-adler.de` is also still WIP and everything currently available under that domain is subject to change, but you can get a first glimpse of our work in the [mvt branch of the maps.gruppe-adler.de repository](https://github.com/gruppe-adler/maps.gruppe-adler.de/tree/mvt).

_I wish these two projects would be further along by now, but this is still just a hobby for us and therefore only gets limited attention 😅_

Thirdly, your output looks wrong but is not really a bug. It's just a technical limitation.
Let me explain:
The simplified process of how grad_mtg generates a tile looks like this:
- open ingame map
- pan / zoom to make sure the desired area covers as much of the screen as possible but is still fully visible
- take a screenshot
- cut screenshot to the area you want to show on the tile
- resize the image to 256x256 pixels

You're using LOD 1, which - as you can [see here](https://github.com/gruppe-adler/grad_mtg#what-is-a-level-of-detail-lod) - corresponds to an area of 12800m x 12800m, but the `worldSize` of Stratis is only 8192m x 8192m. 
Arma 3 maps do not allow to zoom further out than the `worldSize`. Depending on your screen's aspect ratio sometimes even less. (You can try that yourself with the ingame map.)
So basically grad_mtg zooms out as far as possible and takes screenshot, but because the screenshot does not cover the whole area the tile does, the tile gets black borders.

In conclusion: It is not possible to reliably export tiles for LODs which cover a bigger area than the `worldSize`.


Merry Christmas! 🎄

**Comment by Kapppa** [comment]
Well thanks for your response. I will use the new tool. Merry Christmas! 🎄

**Comment by DerZade** [comment]
Perfect :) @Kapppa I just released [a new version of grad_meh](https://github.com/gruppe-adler/grad_meh/releases/tag/0.4rc), so make sure you're using that.
